lib/Client.js

```diff
@@ -869,26 +869,30 @@ class Client extends EventEmitter {
     /**
     * Get the prune count for a guild
     * @arg {String} guildID The ID of the guild
-    * @arg {Number} days The number of days of inactivity to prune for
+    * @arg {Number} [days] The number of days of inactivity to prune for. Default to 7
```

Comment:
`[days=7]` to show default values, not another sentence

Reply:
I thought we would only use those if it defaulted in the code. But I assume it's the same effect anyway.
```suggestion
    * @arg {Number} [days=7] The number of days of inactivity to prune for
```